tests/ddl_reentrant/run.sh

```diff
@@ -33,6 +33,8 @@ ddls=("create database ddl_reentrant" false
       "drop table ddl_reentrant.t2" false
       "recover table ddl_reentrant.t2" false
       "drop database ddl_reentrant" false
+      "alter table ddl_reentrant.t1 add column c1 int, add column c2 int, add column c3 int" false
+      "alter table ddl_reentrant.t1 drop column c1, drop column c2, drop column c3" false
```

Comment:
This case executes these DDLs in sequence, these two DDLs should be before `drop database`